src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/reports/MoveFilterer.kt

```diff
@@ -36,16 +37,18 @@ object MoveFilterer {
   /**
    * A standard move is a completed move with a single completed journey that is billable, the journey from and to
    * destinations should match that of the move, and no cancelled journeys. To be priced as a standard move, the journey
-   * as well as the move must be completed. There also should be no redirects after the move starts, but shouldn't need
-   * to check for this.
+   * as well as the move must be completed. There also should be no redirects after the move starts.
    */
   fun isStandardMove(move: Move) =
-    isCompleted(move) &&
+    move.isCompleted() &&
       with(move.journeys) {
         count { it.stateIsAnyOf(JourneyState.completed) } == 1 &&
           count { isCompleteBillableJourneyAndLocationsMatchMove(it, move) } == 1 &&
           count { it.stateIsAnyOf(JourneyState.cancelled) } == 0
-      }
+      } && when (val moveStart = move.mayBeMoveStartDate()) {
+      null -> true
```

Comment:
So I was in two minds here on line 49 whether to return true or false.  Keeping it true means it will work as before.  Returning false may have undesired side effects.